src/main/java/dev/codesupport/web/domain/validation/validator/ShowcaseValidator.java

```diff
@@ -0,0 +1,54 @@
+package dev.codesupport.web.domain.validation.validator;
+
+import dev.codesupport.web.domain.Showcase;
+import dev.codesupport.web.domain.validation.MultiViolationConstraintValidator;
+import dev.codesupport.web.domain.validation.Violation;
+import dev.codesupport.web.domain.validation.annotation.ShowcaseConstraint;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Validation logic to be performed on properties annotated with {@link ShowcaseConstraint}
+ */
+public class ShowcaseValidator implements MultiViolationConstraintValidator<ShowcaseConstraint, Showcase> {
+
+    /**
+     * Validates a {@link Showcase} object
+     *
+     * @param showcase  The {@link Showcase} object to validate
+     * @param violation Reference to the violation helper object used for this validation
+     */
+    @Override
+    public void validate(Showcase showcase, Violation violation) {
+        if (StringUtils.isBlank(showcase.getTitle())) {
+            violation.missing(Showcase.Fields.title);
+        }
+
+        if (StringUtils.isBlank(showcase.getDescription())) {
+            violation.missing(Showcase.Fields.description);
+        }
+
+        if (StringUtils.isBlank(showcase.getLink())) {
+            violation.missing(Showcase.Fields.link);
+        }
+
+        if (showcase.getContributorList() == null) {
+            violation.missing(Showcase.Fields.contributorList);
+        } else {
+            // If ContributorList exists, validate it
+            ContributorListValidator contributorListValidator = contributorListValidator();
+
+            contributorListValidator.validateContributorList(showcase.getContributorList(), violation, Showcase.Fields.contributorList);
+        }
+    }
+
+    /**
+     * Builds a {@link ContributorListValidator} instance
+     * <p>Exists simply for easier unit testing</p>
+     *
+     * @return A new {@link ContributorListValidator} instance
+     */
+    ContributorListValidator contributorListValidator() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5265305)